### PR TITLE
Dark Mode Optimization

### PR DESCRIPTION
**Pull request by TechWithTyler**
Updated main background and main screen label color to be theme-aware. Dark mode images are same as original light mode images but inverted.

### CODE REVIEW

File: BMI Calculator/Assets.xcassets/calculate_background.imageset/Contents.json [hunk 1]
```diff
@@ -2,22 +2,85 @@
...
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "light"
+        }
+      ],
       "filename" : "calculate_background@3x.png",
+      "idiom" : "universal",
+      "scale" : "3x"
```
Comment (ES-Anon) [inline BMI Calculator/Assets.xcassets/calculate_background.imageset/Contents.json:68]: @TechWithTyler Can you clarify, are these "scale" values used for different higher and lower resolution iPhones?

Comment (TechWithTyler) [reply BMI Calculator/Assets.xcassets/calculate_background.imageset/Contents.json:68]: n times the original image size. I believe the images were built for iPhone but are automatically upscaled for iPad and Mac, and any size variations of devices.
Tip: Supplying every single image size is no longer needed as of Xcode 14--its AI will do it for you!

Comment (ES-Anon) [reply BMI Calculator/Assets.xcassets/calculate_background.imageset/Contents.json:68]: Great, thanks!

**Review by ES-Anon — APPROVED** [review]
Looks great!

**Pull request merged**